### DISCUSSION TITLE
Supported credit cards as logos

### DISCUSSION THREAD

**Discussion by JoseGonzalz** [Telefonica/mistica-design#1881] [Proposals]
### Goal

To display the supported credit cards in forms.

### Context

Currently we need to display the helper text to indicate which credit cards are supported.

### Impact

Show a visual indication of the logos and reducing the spacing in the form.

### Discussion Questions

Do we need to create a credit cards choice option? 
Should we need to create the maximum number of credit card logos displayed?
Should we need to create the visual of unsupported credit cards?

### Do you have a visual proposal?

https://www.figma.com/design/UnI7XlUH58mLvG2QqxEUMe/Explorations-and-Improvements?node-id=309-16759&t=6v6EC6z2srKiIKlo-4

**Comment by AnaMontes11** [comment]
Hi @JoseTelefonica,

We think it's interesting to show a visual indication of the logos, as you suggest, but we believe that including it within the form is outside of the standard (I’ve attached several examples). In cases with more than 3 cards, your proposal doesn’t scale well, as it reduces the space available to display the full credit card number.

In any case, please provide a more contextualized example so we can study it in greater detail.

![a7c55563-aa6c-4655-bd8f-02daeadb981d](https://github.com/user-attachments/assets/41181087-4bb7-4b03-a088-ef149ae49af3)


![0d816096-4974-4320-84ab-c78ac90fff4d](https://github.com/user-attachments/assets/f84d8e7a-0d70-441a-a1ca-13e8d28eeb4d)

**Reply by JoseGonzalz** [reply]
I agree but... I'm afraid we won't have as many credit card options.
Ok, thanks Ana!
